templates/jlink/jlink-builder/build-imagestreams.sh

```diff
@@ -0,0 +1,9 @@
+eval $(crc oc-env)
+VERSION=11
+
+# Buildconfigs attempt to push to ubi9-openjdk-$VERSION-jmods, this needs to exist first
+oc create imagestream ubi9-openjdk-$VERSION-jlink
```

Comment:
Ultimately I hope we don't need a wrapper shellscript, but it is convenient during development.

We could get rid of this line if we include the ImageStream definition in the YAMLs, below. try `oc get -f yaml is/ubi9-openjdk-11-jlink` and then pre-pend the result to `jdk-11-buildconfig.yaml`, separated by `---`.